<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatePur</class>
 <widget class="QWidget" name="TemplatePur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidgetData"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidgetSetNumber">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>原料名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>条形码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>库存</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>原料数量</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>单位</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
         <item>
          <widget class="QLabel" name="labelSize">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButtonOrder">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>生成采购订单</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
